honeybee_radiance_command/pcond.py

```diff
@@ -0,0 +1,83 @@
+"""pcond command."""
+
+from .options.pcond import PcondOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+
+
+class Pcond(Command):
+    """pcond command."""
+
+    __slots__ = ('_input')
+
+    def __init__(self, options=None, output=None, input=None):
+        """Command.
+
+        Args:
+            options: Command options. It will be set to Radiance default values if not
+                provided by user.
+            output: File path to the output file (Default: None).
+            input: File path to the radiance generated hdr file (Default: None).
+        """
+        Command.__init__(self, output = output)
+        if options:
+            self._options = options
+        else:
+            self._options = PcondOptions()
```

Comment:
I started to think we should change these to single-liners but we don't have to!

```suggestion
        self._options = options if options else PcondOptions()
```

Reply:
or actually we can do `self.options = options` and the setter will take care of assigning the default value. That's probably the best option.